packages/embeds/js/src/features/bubble/components/PreviewMessage.tsx

```diff
@@ -67,6 +67,7 @@ const CloseButton = (props: {
 }) => (
   <button
     part="preview-message-close-button"
+    aria-label="preview-message-close-button"
```

Comment:
```suggestion
    aria-label="Close"
```

it should act like a real button label :)